docs/source/integrations/mobile/android-widevine-sdk.md

```diff
@@ -38,70 +40,179 @@ allprojects {
     }
 }		
 ```
-2.	Add your credentials for authentication to the maven repository
+2.	Change the username and password to your credentials for authentication to the maven repository
 
 3.	Under your module’s build.gradle dependencies add:
-___
-```java
- compile 'com.vualto.vudrm:widevine:0.2.0'
-```
 
+	```java
+    implementation 'com.vualto.vudrm:widevine:0.3.4'
+	```
+
+4.	Access to our KID plugin is enabled similarly using the same maven repository configuration, but with the url:
+
+	<https://maven.drm.technology/artifactory/kid-plugin>
+
+	and dependency:
+
+	``` implementation 'com.vualto.vudrm:kidplugin:0.3.4'```
+	
 ## Example Usage
-1.	Instantiate an AssetConfiguration using the fluent interface, minimally you need to provide the content’s KID and DRM token, the default license server in this case will be ‘<https://widevine-proxy.drm.technology/proxy>’. This can also be overridden using the API call ‘licenceUrlWith(<Your license URL>)’.
-```java
-try {
-    assetConfiguration = new AssetConfiguration.Builder()
+
+Instances of VUDRMWidevine are associated with specific assets. Offline assets can be tracked using Exoplayers OfflineLicenseHelper, DownloadAction, DownloadService and DownloadTracker classes. Two types of session are available, determined by the session VUDRM token policy, and the asset configuration.
```

Comment:
Missing oxford comma, should be "Exoplayers OfflineLicenseHelper, DownloadAction, DownloadService, and DownloadTracker classes".